setup.py

```diff
@@ -405,6 +405,10 @@ def setup_package():
         cmdclass=cmdclass,
         python_requires='>=3.7',
         zip_safe=False,
+        setup_requires=['hpy.devel'],
+        # distuils doesn't load hpy.devel unless hpy_ext_modules is present
+        # as a keyword
+        hpy_ext_modules=[],
```

Comment:
@rlamy I'm not sure what the final fix should be yet, but in the mean time this small change made what was happening much more understandable. With it, `--hpy-abi` no longer gives an error for me. I still need to see what actually happens during the build.